pkg/state/context.go

```diff
@@ -103,6 +103,7 @@ type State struct {
 	CredentialsFilePath       string
 	ManifestFilePath          string
 	PauseImage                string
+	ContainerLogsMaxSize      string
```

Comment:
please default this in the constructor, otherwise this will be empty during kubeone install for exmaple.